app/controllers/course/leaderboards_controller.rb

```diff
@@ -30,7 +30,8 @@ def preload_course_levels
 
   # Load approved students from current course with course statistics.
   def load_course_users
-    @course_users = @course.course_users.students.with_approved_state.includes(:user)
+    @course_users = @course.course_users.students.with_approved_state.without_phantom_users.
+                      includes(:user)
```

Comment:
@weiqingtoh the exception to the indentation rule is when an assignment is being made...